tensorboard/webapp/metrics/views/right_pane/saving_pins_dialog/saving_pins_dialog_test.ts

```diff
@@ -0,0 +1,54 @@
+/* Copyright 2024 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+import {CommonModule} from '@angular/common';
+import {TestBed} from '@angular/core/testing';
+import {By} from '@angular/platform-browser';
+import {MatDialogRef} from '@angular/material/dialog';
+
+import {SavingPinsDialogModule} from './saving_pins_dialog_module';
+import {SavingPinsDialogComponent} from './saving_pins_dialog_component';
+
+describe('saving_pins_dialog', () => {
```

Comment:
nit: Using underscores doesn't match the selector (or anything else) and can make the code harder to search for. Suggest: `saving-pins-dialog` or `saving pins dialog`.

(I know some other tests use underscores, but it doesn't seem to be consistent and I think we can adopt a better policy)

Reply:
Applied! Thanks!